src/app/actions/upload.ts

```diff
@@ -0,0 +1,37 @@
+import httpStatusCode from 'http-status-codes';
+
+import { parseCSVFormData } from '@/lib/file';
+import { createSearchResult } from '@/repositories/searchResult';
+
+export async function handleCsv(file: FormDataEntryValue | null, userId: number) {
```

Comment:
Naming-wise, what do you think of the following:

```suggestion
export async function uploadCsv(file: FormDataEntryValue | null, userId: number) {
```

It is simple and correspond more or less to teh file name. I do not see value in having the prefix `handle` 🤔